Source/Charts/Data/Implementations/Standard/LineChartDataSet.swift

```diff
@@ -13,8 +13,12 @@ import Foundation
 import CoreGraphics
 
 
-open class LineChartDataSet: LineRadarChartDataSet, ILineChartDataSet
+open class LineChartDataSet: LineRadarChartDataSet, ILineChartDataSet, IValueSliceLineChartDataSet
 {
+    open var drawCirclesOnlyForHighlightedValueEnabled = false
```

Comment:
The idea for having both of these, is for objective c syntax, the simple `drawCircles...Enabled` variable should have `@objc` before it while the `isDraw...Enabled` shouldn't